### ISSUE TITLE
Some of the operators in the model are not supported by the standard TensorFlow Lite runtime. If those are native TensorFlow operators, you might be able to use the extended runtime by passing --enable_select_tf_ops, or by setting target_ops=TFLITE_BUILTINS,SELECT_TF_OPS when calling tf.lite.TFLiteConverter(). Otherwise, if you have a custom implementation for them you can disable this error with --allow_custom_ops, or by setting allow_custom_ops=True when calling tf.lite.TFLiteConverter(). Here is a list of builtin operators you are using: ABS, ADD, BATCH_TO_SPACE_ND, CAST, CONCATENATION, CONV_2D, EXPAND_DIMS, GREATER, LESS, LOGICAL_AND, LOGICAL_NOT, MAXIMUM, MAX_POOL_2D, MINIMUM, MUL, NEG, PACK, PAD, RELU, RESHAPE, RSQRT, SELECT, SOFTMAX, SPACE_TO_BATCH_ND, SQUARE, STRIDED_SLICE, SUB, SUM, TOPK_V2. Here is a list of operators for which you will need custom implementations: FIFOQueueV2, QueueDequeueV2, SparseSoftmaxCrossEntropyWithLogits.

### ISSUE THREAD

**Issue by laiou** [tensorflow/tensorflow#33621]
**System information**
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04):
- TensorFlow installed from (source or binary):
- TensorFlow version (or github SHA if from source):

**Provide the text output from tflite_convert**

```
# Copy and paste here
```

Also, please include a link to a GraphDef or the model if possible.

**Any other info / logs**

Include any logs or source code that would be helpful to diagnose the problem. If including tracebacks, please include the full traceback. Large logs and files should be attached.

**Comment by ymodak** [comment]
As the error message suggested you will need custom implementations for following operations;
```FIFOQueueV2, QueueDequeueV2, SparseSoftmaxCrossEntropyWithLogits..```

- To make a custom op you may refer https://www.tensorflow.org/lite/guide/ops_custom
- You can also raise a request to add these features in TF by commenting on this issue thread.
  https://github.com/tensorflow/tensorflow/issues/21526
- You can try converting your model using ```TFLITE_BUILTINS, SELECT_TF_OPS``` flags to reduce custom implementation of ops (only a subset of ops can be implemented by this method while others may still require custom implementation )
